app.js

```diff
@@ -25,16 +25,16 @@ app.get("/", (req, res) => {
   console.log(process.env.JWT_KEY);
 });
 
-app.all("*", (req, res, next) => {
-  next(new ExpressError(404, "Page not found!"));
-});
+// app.all("*", (req, res, next) => {
+//   next(new ExpressError(404, "Page not found!"));
+// });
 
-app.use((err, req, res, next) => {
+app.use((err, req, res) => {
```

Comment:
This is error handling middleware, ```next``` is neccessary.

Reply:
added back